packages/cozy-harvest-lib/src/locales/en.json

```diff
@@ -158,6 +158,18 @@
     "administrative": "Administrative",
     "photos": "Photos"
   },
+  "infos": {
```

Comment:
Nit: I would remove the `infos`, it doesn't really make sense in the context of translations (like other existing translation keys, but might as well not pile them on)

Reply:
That's a personal habit and taste, I have been using namespaces in locale keys for years. In the same file there is also `cards` and `errors`. I don't mind to change it, the thing is we really never statued a way to name our locale keys.

Reply:
Yeah fair enough. I just find that when we move the titles etc outside of the `infos` component, suddenly the translation key looks super weird. But it's really a detail, I don't mind keeping things as they are in the PR :)

Reply:
Maybe I am a little bit psychorigid here, but in my opinion if the title is move its key must change. In my mind a locale key is also a treasure map about where to find my translation in the app. By the way I could be wrong, I will make a few research about best practices and arguments about locale keys naming conventions :)